modeler.py

```diff
@@ -33,22 +39,23 @@ def __call__(self, state, action):
         Args:
           state (object): Current world state to start from.
           action (object): Action to take in current world state.
+
         None action is not allowed.
 
         Returns:
           (object): Next state.
           (float): Reward or final state value.
           (bool): Is terminal.
 
-          or
-
-          If action is not allowed, returns (None, None, None).
+        If action is not allowed, returns (None, None, None).
```

Comment:
Tak jest gorzej niż poprzednio moim zdaniem. To należy to "Return:" więc na pewno wymaga wcięcia i niech wtedy przylega do tamtych. Ale tak jak było wcześniej, to chociaż było wiadomo, że albo tamte trzy wartości albo trzy razy None.
Można też dodać to if za każdym zwracanym obiektem:
```
Returns:
    (object): Next state or None if action is not allowed.
    (float): Reward or final state value or None if action is not allowed.
    (bool): Is terminal or None if action is not allowed.
```